app/components/file-details/vulnerability-analysis-details/index.scss

```diff
@@ -61,19 +65,27 @@
         }
       }
     }
-  }
 
-  .analysis-section {
-    width: 100%;
-    padding: 1.5em;
+    &.business-implication-section {
+      border: 1px solid var(--file-details-vulnerability-analysis-details-divider-color);
+    }
 
-    &:not(:last-child) {
+    .analysis-content-description-title {
+      width: 100%;
+      padding: 1em;
       border-bottom: 1px solid
+            var(--file-details-vulnerability-analysis-details-divider-color);
+    }
+
+    &:not(:last-child) {
+      border: 1px solid
         var(--file-details-vulnerability-analysis-details-divider-color);
     }
 
     &.analysis-regulatory-section {
       padding: 0em;
+      border: 1px solid
+            var(--file-details-vulnerability-analysis-details-divider-color);
```

Comment:
can we rename this to `--file-details-vulnerability-analysis-details-border-color`